app.js

```diff
@@ -69,6 +71,18 @@ app.use(function(req, res, next){
   }
 })
 
+app.use(function(req, res, next){
+  console.log(req.hostname, res.headers)
```

Comment:
Thanks for adding this - just a minor gotcha:
```suggestion
  console.log(req.hostname, req.headers)
```